docs/docs/getting-started/02-oauth-tutorial.mdx

```diff
@@ -125,17 +141,18 @@ export default function CamperVanPage() {
 }
 ```
 
-You can use the `useSession` hook from anywhere in your application (e.g. in a header component). Behind the scenes, the hook will connect to the `<SessionProvider />` to read the current user session.
+You can use the `useSession` hook from anywhere in your application (E.g. in a header component). Behind the scenes, the hook will connect to the `<SessionProvider />` to read the current user session.
 
 ### Protecting API Routes
 
-Protecting your custom API Routes (.i.e not allowing a resource to be accessed in case the user is not logged in) is easy! You can use [`getSession()`](/reference/utilities/#getsession) to know whether a session exists or not:
+Protecting your API Routes (blocking unauthorized access to resources) is easy! You can use [`getServerSession()`](/reference/nextjs#getserversession) to know whether a session exists or not:
 
 ```ts title="pages/api/movies/list.ts"
-import { getSession } from "next-auth/react"
+import { getServerSession } from "next-auth/next"
+import authOptions from "@/pages/api/auth/[...nextauth]"
```

Comment:
```suggestion
import authOptions from "../auth/[...nextauth]"
```

Let's not assume anything about the user's setup. They can change it if they want to.